### DISCUSSION TITLE
Mobile apps [Sep 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6515] [Discord Threads]
> 🙋‍♂️ **yoitstrip**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### morning morning 🙂 Thank you   
#### We do've plans in the future.   
####    
#### What has been your usecase with nocodb ?  
#  
> 🙋‍♂️ **yoitstrip**  
#### So far I'm just exploring AirTable alternatives, I do a lot of relational data modeling for psychiatric patients by monitoring trends in therapy notes (repeated phrases, keywords tied to emotions, cognitive fallacies, etc) and their own journal entries. We've recently partnered with local crisis teams to deploy counselors in the field and want a self-hostable (and thus secure) method of allowing our relational databases to be available to these teams in real time  
#  
> 🙋‍♂️ **yoitstrip**  
#### Right now it's all on google sheets and we don't love that AirTable a) is expensive and b) is closed source and if we miss a payment or need to migrate, client health can severely suffer as a result  
#  
> 🙋‍♂️ **yoitstrip**  
#### And of course, sheets is less then ideal for a myriad of reasons  
#  
> 💁 **o1lab**  
#### cool, thanks for the details  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1143215013222424646)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
